src/pages/students/forms/paymentDataForm/PaymentData.tsx

```diff
@@ -49,62 +49,72 @@ const PaymentData = forwardRef<FormProp<PaymentDataProps>, FormProp<PaymentDataP
   } = useForm<FormData>({
     resolver: yupResolver(paymentDataScheme) as any,
     defaultValues: { ...studentRegistrationForm },
+    context: {
+      actionType: actionType,
+    }
   });
 
   const PaymentPlanNameSelected = watch("paymentPlanName");
 
   const NoPaymentSelectedSkeleton = () => {
     return (
       <div>
-        <Text>Este campo se habilitará una vez seleccione el plan de pago</Text>
+        <Text $isEdit={true}>Este campo se habilitará una vez seleccione el plan de pago.</Text>
```

Comment:
Esto no debería depender del campo "actionType"? algo como:

```suggestion
        <Text $isEdit={actionType === 'edit'}>Este campo se habilitará una vez seleccione el plan de pago.</Text>
```